<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MumeProtocolPage</class>
 <widget class="QWidget" name="MumeProtocolPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>305</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Mume Client Protocol Features</string>
     </property>
     <layout class="QGridLayout">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="1" column="0">
       <widget class="QCheckBox" name="xmlSupportCheckBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>XML</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Remote Editing</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_1">
      <item row="3" column="0">
       <widget class="QRadioButton" name="externalEditorRadioButton">
        <property name="text">
         <string>External editor:</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="externalEditorCommand"/>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="externalEditorBrowseButton">
        <property name="text">
         <string>&amp;Browse</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QRadioButton" name="internalEditorRadioButton">
        <property name="text">
         <string>Internal editor</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>341</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>xmlSupportCheckBox</tabstop>
  <tabstop>internalEditorRadioButton</tabstop>
  <tabstop>externalEditorRadioButton</tabstop>
  <tabstop>externalEditorCommand</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
